plugins/org.jkiss.dbeaver.ext.hana/src/org/jkiss/dbeaver/ext/hana/model/HANASysViewColumn.java

```diff
@@ -23,6 +23,10 @@
 
 public class HANASysViewColumn extends HANATableColumn {
 
+	// boolean like columns in SYS schema are stored as VARCHAR(5) with values TRUE, FALSE
+    static final String TRUE = "TRUE";
+    static final String FALSE = "FALSE";

```

Comment:
Perhaps we should move these constants into their own class, `HANAConstants`?

It should be marked as `final` and contain a `private` constructor since it's a utility class.

There are also several usages of these constants in the `HANAProcedure` class.